webapp/components/Surveys/index.js

```diff
@@ -0,0 +1 @@
+export { Surveys } from './Surveys'
```

Comment:
I would move this under webapp/components/survey/Surveys/

Reply:
ok